icdiff

```diff
@@ -535,8 +535,12 @@ def diff(options, a, b):
     is_b_file = not os.path.isdir(b)
 
     if is_a_file and is_b_file:
-        if not filecmp.cmp(a, b, shallow=False):
-            diff_files(options, a, b)
+        try:
+            if not filecmp.cmp(a, b, shallow=False):
+                diff_files(options, a, b)
+        except OSError as e:
+            if e.errno == errno.ENOENT:
+                print_meta("error: file '%s' was not found" % e.filename)
```

Comment:
This will swallow any `OSError` exceptions where the `errno` isn't `ENOENT`.  We should either print or raise in response to any errors.